src/components/TogglePublishPopup/TogglePublishPopup.tsx

```diff
@@ -19,16 +19,25 @@ accepting all external messages!`
   const unpublishedText = `Your profile is now not visible to the public.
 You can republish it at anytime.`
 
+  const pendingText = `Your profile is currently under review by our administration team. 
+We will notify you once the review is complete. Thank you for your patience.`
+
   return (
     <div className={styles.overlay}>
       <div className={styles.container}>
         <h2 className={styles.header}>
           {state === PublishingState.APPROVED
             ? 'Congratulations'
-            : 'Your profile’s unpublished'}
+            : state === PublishingState.PENDING
```

Comment:
**nitpick:** instead of duplicating these equations, we could do something like to encapsulate all profile related verification logic: 
```ts
// data/frontend/profile/profile.helpers.ts
ProfileVerification.isPending(state)
ProfileVerification.isApproved(state)
ProfileVerification.isRejected(state)
```